src/main/java/io/swagger/codegen/languages/kotlin/AbstractKotlinCodegen.java

```diff
@@ -584,4 +588,16 @@ protected boolean needToImport(String type) {
 
         return imports;
     }
+
+    @Override
+    public CodegenProperty fromProperty(String name, Schema propertySchema) {
```

Comment:
Is this required? or is it simply enough to set the typemapping for array as you have done above?

Reply:
The containerType is yet to be referred at any Kotlin mustache templates.  However ,  at the DefaultCodegenConfig class,  line 51 has assigned the containerType = ''array"  if it is the swagger ArraySchema. This is due to the share codebase with other languages ( some java mustache templates has a reference to containerType.  To make it consistent with the conversion to List for Kotlin,  I think would be better re-assign the containerType = "list" .   
 
Anyway , some part of codes at DefaultCodegenConfig marked both Array and List with this constant IS_LIST_CONTAINER_EXT_NAME  . It appeared to me array and list are inter-changeably at this context .